packages/core/src/shared/errors.ts

```diff
@@ -391,7 +390,9 @@ export function scrubNames(s: string, username?: string) {
             }
         }
 
-        r += ` ${start.replace(/\\/g, '/')}${scrubbed.replace(/^[\/\\]+/, '')}${fileExt}`
+        // includes leading '.', eg: '.json'
+        const fileExt = path.extname(pathSegments[pathSegments.length - 1]) ?? ''
+        r += ` ${start.replace(/\\/g, '/')}${scrubbed.replace(/^[\/\\]+/, '')}${fileExt.length < 6 ? fileExt : ''}`
```

Comment:
do we have a test case for the `length>=6` case?

Reply:
Added in